resources/performance.go

```diff
@@ -13,15 +13,5 @@ func Performance() *schema.Table {
 		Resolver:  fetchPerformance,
 		Transform: transformers.TransformWithStruct(rest.Performance{}),
 		Multiplex: client.ProjectMultiplex,
-		Columns: []schema.Column{
-			{
-				Name:     "project_id",
```

Comment:
Removing as https://docs.swetrix.com/statistics-api#get-v1logperformance doesn't return a project_id or any unique key in the response